app/src/main/res/layout/activity_drupal_login.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="#5fb0c9"
+                android:orientation="vertical"
+                android:id="@+id/drupal_login"
+                android:scrollbarAlwaysDrawVerticalTrack="true">
+    <include
+        android:id="@+id/toolbar"
+        layout="@layout/toolbar"/>
+
+    <TextView
+        android:id="@+id/login_title"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:layout_marginTop="22dp"
+        android:gravity="center_horizontal"
+        android:text="Account Login"
+        android:textColor="@color/white"
+        android:textSize="26sp"
+        android:layout_below="@+id/toolbar"
+        android:textStyle="bold" />
+
+    <RelativeLayout
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/login_title"
+        android:layout_marginLeft="30dp"
+        android:layout_marginRight="30dp"
+        android:layout_marginTop="70dp"
+        android:background="@color/white"
+        android:elevation="4dp"
```

Comment:
@Subhankar29 there are hardcoded dimensions in lot of places. Avoid using much of them for best view in all sizes of the screen. Also add them in `dimen.xml`